lld/wasm/SymbolTable.cpp

```diff
@@ -612,10 +623,11 @@ Symbol *SymbolTable::addUndefinedGlobal(StringRef name,
   if (s->traced)
     printTraceSymbolUndefined(name, file);
 
-  if (wasInserted)
+  if (wasInserted) {
+    traceImport(importName, file);
```

Comment:
`traceImport` sounds like telling the program to start tracing the import, which doesn't seem to be the case; it just prints the import. 

By the way, I'm not very familiar with the code structure here so I might be missing something, but should we call `printTraceSymbolUndefined` and `traceImport` separately from several places? Can we do both in one function and preferably one place, when `importName` is not null?